root-mode for babel-node

I'm using Babel 7 and I have a monorepo with lerna and yarn workspaces, I could configure the script like this: `babel src --root-mode upward --out-dir dist --delete-dir-on-start` and it works really nice, however I want to use `nodemon` with `babel-node` but I noticed that I can't specify `--root-mode` for `babel-node`, what can I do to solve that?

`nodemon --legacy-watch --exec babel-node -- src/index`

I'm using 7.2.0 but still not working.
`babel-node --root-mode upward src/index.js`

> internal/modules/cjs/loader.js:605
>    throw err;
>    ^
>
> Error: Cannot find module 'C:\Users\ZD\Desktop\acme\packages\server\upward'

@loganfsmyth 

> yarn babel-node --root-mode upward src/index.js

> internal/modules/cjs/loader.js:605
> throw err;
> Error: Cannot find module 'C:\Users\ZD\Desktop\acme\packages\server\upward'

It's trying to load `upward` as a module

--- a/packages/babel-node/src/_babel-node.js
+++ b/packages/babel-node/src/_babel-node.js
@@ -52,7 +52,7 @@ program.option(
   "The name of the 'env' to use when loading configs and plugins. " +
     "Defaults to the value of BABEL_ENV, or else NODE_ENV, or else 'development'.",
 );
-commander.option(
+program.option(
   "--root-mode [mode]",
   "The project-root resolution mode. " +
     "One of 'root' (the default), 'upward', or 'upward-optional'.",

